components/Finance/FundCard.tsx

```diff
@@ -0,0 +1,171 @@
+      })
+      .join(' ');
+
+    const gradientOffsets = [0, 50, 100];
+
+    return { polyline: polylinePoints, gradientStops: gradientOffsets };
+  }, [points]);
+
+  if (!points.length) return <div className={styles.sparklinePlaceholder}>数据准备中</div>;
+
+  const gradientId = `${chartId}-gradient`;
+
+  return (
+    <div className={styles.sparkline}>
+      <svg viewBox="0 0 100 40" role="img" aria-label="近 60 日走势">
+        <defs>
+          <linearGradient id={gradientId} x1="0%" y1="0%" x2="100%" y2="0%">
+            {gradientStops.map(offset => (
+              <stop
+                key={offset}
+                offset={`${offset}%`}
+                stopColor="var(--bs-primary)"
+                stopOpacity="0.5"
+              />
+            ))}
+          </linearGradient>
+        </defs>
+        <polyline
+          fill="none"
+          stroke={`url(#${gradientId})`}
+          strokeWidth="2"
+          strokeLinejoin="round"
+          strokeLinecap="round"
+          points={polyline}
+        />
+      </svg>
+    </div>
+  );
+};
+
+export const FundCard: FC<FundCardProps> = ({ data }) => {
+  const {
+    displayName,
+    category,
+    riskLevel,
+    latestValue,
+    dailyChangePct,
+    oneYearReturnPct,
+    maxDrawdownPct,
+    tags,
+    sparkline,
+    updatedAt,
+    fallback,
+    description,
+  } = data;
+  const sparklineId = useMemo(() => `fund-${data.symbol}`, [data.symbol]);
+
+  return (
+    <Card className={`${styles.fundCard} h-100`}>
+      <Card.Body className="d-flex flex-column gap-3">
+        <div
+          className={`${styles.fundCardHeader} d-flex justify-content-between align-items-start`}
+        >
+          <div>
+            <h3 className="h5 mb-1">{displayName}</h3>
+            <div className="d-flex flex-wrap gap-2 align-items-center">
+              <Badge bg="light" text="dark">
+                {INDEX_CATEGORY_LABELS[category]}
+              </Badge>
+              <Badge
+                bg={
+                  riskLevel === 'aggressive'
+                    ? 'danger'
+                    : riskLevel === 'balanced'
+                      ? 'warning'
+                      : 'success'
+                }
+              >
+                {INDEX_RISK_LABELS[riskLevel]}
+              </Badge>
+              {fallback && (
+                <Badge bg="secondary" text="light">
+                  离线数据
+                </Badge>
+              )}
+            </div>
+          </div>
+          <small className="text-muted text-end">
+            数据源 <br />
+            {data.source.historyEndpoint}
+          </small>
+        </div>
+
+        <p className="text-muted mb-0">{description}</p>
+
+        <div className={`${styles.metricRow} d-flex flex-wrap gap-4 align-items-center`}>
+          <div>
+            <p className="text-muted mb-1">最新点位</p>
+            <span className={styles.metricValue}>{formatNumber(latestValue)}</span>
+          </div>
```

Comment:
```suggestion
        <dl className={`${styles.metricRow} d-flex flex-wrap gap-4 align-items-center`}>
            <dt className="text-muted mb-1">最新点位</dt>
            <dd className={styles.metricValue}>{formatNumber(latestValue)}</dd>
```
https://developer.mozilla.org/zh-CN/docs/Web/HTML/Reference/Elements/dl